ocsci/enums.py

```diff
@@ -1,8 +1,8 @@
 from enum import Enum
 
 
-class TestStatus(Enum):
-    # As we have function to transform RC to TestStatus Enum we should add new
+class StatusOfTest(Enum):
```

Comment:
Do we even need this enum anymore if we are utilizing pytest? I think tests should be asserting anything they need as passing criteria and we should let pytest handle the test status on it's own. Thoughts?

Reply:
We don't need them ;) I just didn't want to deal with cleanup of everything yet till we have pytest ready. There was issue with Test in the name of class and pytest tried to execute it. So I used easiest way for me and just renamed all occurrences for now to prevent pytest to consider it as test ;)